### PR TITLE
Plans of Action

### LINKED ISSUE #26: Plans of Action
- [ ] View current and past plans
- [ ] Add things
- [ ] Edit things

### PR DESCRIPTION
**Pull request by Joeyh021**
Added two new templates - Plan, and new plan.
Both templates work for either mentor/mentee, based off stuff passed to the template by the view. Both mentees and mentors can view and edit plans of actions using a form (which needs a bit of styling yet).

There's probably all sorts of little django fuckups in here so please pull this apart.

### CODE REVIEW

File: people/forms.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
-from .models import User
+from .models import User, BusinessArea, Topic, UserType
 
 
 class RegistrationForm(UserCreationForm):
     email = forms.EmailField(required=True)
     first_name = forms.CharField(label="First name", max_length=100)
     last_name = forms.CharField(label="Last name", max_length=100)
+    business_area = forms.ModelChoiceField(
```
Comment (joshdavies14) [inline people/forms.py]: Is there a reason you've added business area here? (and in the meta class?) - it doesn't appear in the registration form, it is done on the "edit profile" page which we redirect to once the account is created, and unless I am oblivious there isn't an update to the registration form here?

**Review by joshdavies14 — APPROVED** [review]
This was just a quick glance but besides my one query it all lgtm :)

File: people/forms.py (hunk 1 above)
Comment (Joeyh021) [reply people/forms.py]: oh i forgot i did this, meant to revert it. I was trying to enforce that a user object cannot be created with no business area, and I cant remember why. I'll fix this up later and check everything still works

Comment (joshdavies14) [reply people/forms.py]: no worries! :)

**Pull request merged**